src/pages/overlay/components/activationButtons/activationButtons.module.scss

```diff
@@ -1,16 +1,21 @@
 @import '../../../../variables.scss';
 
 .activationButtons{
-    margin-top: 40px;
+    margin-top: calc(6 * $overlay-base-size); //move slightly lower relative to available height
```

Comment:
Can this adjusted margin be copied over to overlaysettings to make sure they stay aligned?

Reply:
yes sure. it is aligned at the top now, but otherwise it does not scale. i guess thats fine? the toggle button should not get any smaller, right?

Reply:
Thanks! That seems fine to me, I can't imagine scaling it will be useful (maybe at a certain point it should disappear?) If more settings are added down the road maybe it collapses into another menu like the list, but that's a future problem.